doc/related_projects.rst

```diff
@@ -80,9 +64,6 @@ enhance the functionality of scikit-learn's estimators.
 - `mlxtend <https://github.com/rasbt/mlxtend>`_ Includes model visualization
   utilities.
 
-- `scikit-plot <https://github.com/reiinakano/scikit-plot>`_ A visualization library
```

Comment:
@rth can we merge in sklearn-evaluation.
1. It will allow support in backward compatability.
2. Still well maintained, has tons of new features and bug fixes.